libs/build_params.py

```diff
@@ -29,11 +30,27 @@ def get_interventions(start_date=datetime.now().date()):
         },
     ]
 
+
 OUTPUT_DIR = 'results/test'
-OUTPUT_DIR_COUNTIES = 'results/county'
+COUNTY_SUBDIR = 'county'
+COUNTY_SUMMARY_SUBDIR = 'county_summaries'
+# Output directory for latest case data by state
+STATE_SUMMARY_SUBDIR = "case_summary"
```

Comment:
Rather than hardcoding all of this in python-land, can we make the python scripts accept an arbitrary output directory?  i.e.

run_model.py --output ../covid-projections/public/data/

FWIW- We are probably going to tweak this directory structure as part of cleaning up our "API"...

Reply:
+1 this

Reply:
yeah I like this too.